alf/algorithms/hypernetwork_algorithm.py

```diff
@@ -132,6 +136,13 @@ def __init__(self,
             num_particles (int): number of sampling particles
             entropy_regularization (float): weight of entropy regularization
 
+            Args for the critic (used when par_vi is 'minmax')
```

Comment:
Take a look at the compiled documentation to make sure it display correctly.
And see https://alf.readthedocs.io/en/latest/notes/preview_doc_website.html for viewing the doc locally.